src/main/java/com/github/awwkoala/selenium/page/DropDownPage.java

```diff
@@ -0,0 +1,70 @@
+package com.github.awwkoala.selenium.page;
+
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.List;
+
+public class DropDownPage extends BasePage {
+
+  @FindBy(id = "dropdown")
+  private WebElement dropdown;
+
+
+  @FindBy(css = "#dropdown option")
+  private List<WebElement> options;
+
+  public DropDownPage(WebDriver driver) {
+    super(driver);
+  }
+
+  @Override
+  public void goTo() {
+    driver.get("http://theinternet.przyklady.javastart.pl/dropdown");
+  }
+
+  private void selectOption(int i) {
```

Comment:
You can define `DropDownPage` as a return type instead of void, then you can use it as inline return